src/app/comic/comic-list-container/comic-list-container.component.html

```diff
@@ -1,11 +1,27 @@
 <mu-header></mu-header>
-<div>
-  <button id = "add-comic">Add Comic</button>
+<div class="button-wrapper">
+  <button id="add-comic" (click)="onComicModalClick()">Add Comic</button>
 </div>
+<section class="modal-container" *ngIf="showComicModal">
+  <form (ngSubmit)="addComicDetail()" [formGroup]="modalComicFormGroup">
+    <div class="input-wrapper">
+      <input required class="modal-input-field" type="number" id="comicId" placeholder="Enter ComicID" formControlName="comicId">
```

Comment:
also id need not be an input, it could be hardcoded based on how many results are there